fix(docs): unify sequencer / validator naming

sklppy88 opened the pull request:
This does not touch any CLI option naming. this should be done by eng / dev rel with confirmation from eng in a follow up pr

critesjosh commented on docs/docs/the_aztec_network/guides/run_nodes/how_to_run_sequencer.md:
```diff
@@ -66,8 +66,8 @@ The following variable names are specific to the `aztec start` command, set them
-- `VALIDATOR_PRIVATE_KEY="Ox<hex value>"`: Private key of testnet L1 EOA that holds Sepolia ETH (0.01 Sepolia ETH can get you started)
-- `COINBASE="0x<eth address>"`: Recipient of block rewards (for node security on mainnet, this should be a different address to the validator eoa)
+- `VALIDATOR_PRIVATE_KEY="Ox<hex value>"`: Private key of testnet L1 EOA that defines the sequencer identity that holds Sepolia ETH (0.01 Sepolia ETH can get you started)
+- `COINBASE="0x<eth address>"`: Recipient of block rewards (for node security on mainnet, this should be a different address to the sequencer eoa)
```
```suggestion
- `COINBASE="0x<eth address>"`: Recipient of block rewards (for node security on mainnet, this should be a different address to the sequencer EOA)
```

critesjosh commented on docs/docs/the_aztec_network/reference/cli_reference.md:
has the CLI been updated, or is it being updated? If not, maybe we shouldn't change things on this page?

sklppy88 replied:
My thought process was to only changing the descriptions in this file, and the actual CLI description references can be changed later. But yes I see your point and have removed changes to these files to make sure there is always parity even in descriptions !

critesjosh commented on docs/versioned_docs/version-v1.2.0/the_aztec_network/guides/run_nodes/how_to_run_sequencer.md:
```diff
@@ -66,8 +66,8 @@ The following variable names are specific to the `aztec start` command, set them
-- `VALIDATOR_PRIVATE_KEY="Ox<hex value>"`: Private key of testnet L1 EOA that holds Sepolia ETH (0.01 Sepolia ETH can get you started)
-- `COINBASE="0x<eth address>"`: Recipient of block rewards (for node security on mainnet, this should be a different address to the validator eoa)
+- `VALIDATOR_PRIVATE_KEY="Ox<hex value>"`: Private key of testnet L1 EOA that defines the sequencer identity that holds Sepolia ETH (0.01 Sepolia ETH can get you started)
+- `COINBASE="0x<eth address>"`: Recipient of block rewards (for node security on mainnet, this should be a different address to the sequencer eoa)
```
```suggestion
- `COINBASE="0x<eth address>"`: Recipient of block rewards (for node security on mainnet, this should be a different address to the sequencer EOA)
```

critesjosh commented on docs/versioned_docs/version-v1.2.0/the_aztec_network/reference/cli_reference.md:
Probably shouldnt change this page, since this code hasn't changed.

sklppy88 committed:
fix(docs): unify sequencer / validator naming

This does not touch any CLI option naming. this should be done by eng / dev rel with confirmation from eng in a follow up pr

Co-authored-by: Esau <esau@aztecprotocol.com>
